server/package.json

```diff
@@ -22,6 +22,13 @@
         "test:e2e": "jest --config ./test/jest-e2e.json --runInBand",
         "checks": "npm run && npm run test && npm run lint",
         "prepare": "cd .. && husky install && cd server",
+        "typeorm": "ts-node ./node_modules/typeorm/cli",
+        "typeorm:migration-run": "npm run typeorm migration:run -- -d ./src/datasource.ts",
+        "typeorm:migration-generate": "npm run typeorm -- -d ./src/datasource.ts migration:generate ./migrations/$npm_config_name",
+        "typeorm:migration-create": "npm run typeorm -- migration:create ./migrations/$npm_config_name",
+        "typeorm:migration-revert": "npm run typeorm -- -d ./src/datasource.ts migration:revert",
+        "typeorm:schema-drop": "npm run typeorm -- -d ./src/datasource.ts schema:drop",
+        "typeorm:schema-sync": "npm run typeorm -- -d ./src/datasource.ts schema:sync",
         "seed:drop": "ts-node -r tsconfig-paths/register src/database/seeding/seed.ts"
```

Comment:
While we're looking at scripts, could you rename the `seed:drop` command to another name similar to what you named the other commands and also update README.md